### ISSUE TITLE
Feature Request: Move  Fodling Arrows to the right

### ISSUE THREAD

**Issue by chrisgrieser** [kevboh/longform#140]
**Describe the solution you'd like**
How it currently looks:
![Pasted image 2023-01-24 18 46 18](https://user-images.githubusercontent.com/73286100/214368804-8217883d-afbf-48ae-b92a-ee93b5da87c3.png)

How it looks with the arrows switched. Notice how 2. and 3. are now properly aligned.
![Pasted image 2023-01-24 18 45 48](https://user-images.githubusercontent.com/73286100/214368692-6f0f02c1-d031-40c0-adc6-12578fe947fe.png)

**Describe alternatives you've considered**
I used css to fix this, though a proper solid would obviously be preferable.

**Comment by colintedford** [comment]
This is a problem in Obsidian's sidebars in general (see the file browser and the outline). There's a [feature request](https://forum.obsidian.md/t/outline-formatting-is-misleading-misaligned-when-only-some-nodes-have-children/42975) to fix it and a [help thread](https://forum.obsidian.md/t/outline-alignment-in-recent-updating-is-bit-of-confusing/50522/12) containing CSS that might fix it (I haven't read thru and tried it yet).

IMO it would be better to fix the alignment so the arrow sits outside the outline on the left than to move it to the right. That would keep it consistent with how headings and lists are, and with how operating systems style file trees. I also think it's easier to see which items can be folded/unfolded when scanning the list.

**Comment by vishae** [comment]
@chrisgrieser if you could share the css you created to fix the alignment issue, that would be great!

**Comment by chrisgrieser** [comment]
@vishae I use this
```css
body #scene-list .scene-container {
	display: flex;
	flex-direction: row-reverse;
}

body #scene-list .scene-container div {
	margin-right: 4px;
	margin-left: 0;
}
```

**Comment by vishae** [comment]
@chrisgrieser thanks!

**Comment by vishae** [comment]
@chrisgrieser not sure if this is happening to you, but now when I use the css snippet, the indented sections don't fold up anymore. The arrows still flips to show that the section should be folded/unfolded, but nothing actually happens.

**Comment by chrisgrieser** [comment]
@vishae yeah, either on Obsidian's side or on the Longform plugin's side something changed. I think for a stable solution, we would need to have a proper implementation from the Longform plugin.
